src/parsers/handlers/marshall/MarshallConfigHandler.ts

```diff
@@ -31,7 +32,11 @@ export class MarshallConfigHandler extends AbstractYamlHandler {
     }
 
     loadDefaultConfig<K extends keyof LocalSettings>(key: K, value: Literal, localSettings: LocalSettings): LocalSettings {
-        localSettings[key] = value as any;
+        var escapedValue = value
+        if (typeof value === "string") {
```

Comment:
Dataview has a WrappedLiteral where you can obtain the type. This should work too, but could be interesting to use it
(examples in dataviewService.ts)

Reply:
also. Could you use `let`instead of `var`? rollup build optimisation